client/GameComponents/Messages.jsx

```diff
@@ -133,6 +140,10 @@ class InnerMessages extends React.Component {
                 return (
                     <span className={ 'icon-clan-' + fragment } key={ index++ } ><span className='hide-text'>{ fragment }</span></span>
                 );
+            } else if(this.otherIcons[fragment]) {
+                return (
+                    <img className={ this.otherIcons[fragment].className } key={ index++ } src={ this.otherIcons[fragment].imageSrc }/>
```

Comment:
Might be worth adding a title property to ease people into the change (so they get a tooltip when hovering over).

```html
<img className={ this.otherIcons[fragment].className } title={ this.otherIcons[fragment].title } key={ index++ } src={ this.otherIcons[fragment].imageSrc }/>
```